ui/jobs/jobs.js

```diff
@@ -681,9 +681,12 @@ class Bars {
   }
 
   _onPartyWipe(e) {
-    // TODO: add reset for job-specific ui
     if (this.buffTracker)
       this.buffTracker.clear();
+    // Reset job-specific ui
```

Comment:
This is too many things in one PR.  Can you remove this bit and submit another PR? The other changes are fine, but I think resetting timers is not the same as resetting all job ui.  It doesn't clear resources or any other things.  I think it'd be better if each job had a `reset()` function to clean up anything it added.  However, I think that's too many changes for this PR and so it'd be better to do separately.

Reply:
Reverted. I'll try to write a `reset()` function later.